vmdb/lib/evm_database.rb

```diff
@@ -17,9 +17,9 @@ class EvmDatabase
     VmdbDatabase
   }
 
-  def self.model_class_names
-    @model_class_names ||= begin
-      Dir.glob(File.join(Rails.root, "app/models/*.rb")).collect { |f| File.basename(f, ".*").camelize }.compact.sort
+  def self.seedable_model_class_names
+    @seedable_model_class_names ||= begin
+      Dir.glob(Rails.root.join("app/models/*.rb")).collect { |f| File.basename(f, ".*").camelize if File.read(f).include?("self.seed") }.compact.sort
```

Comment:
Note that seeding only happens for *some* workers, so that last statement is not entirely accurate.